test cases/windows/8 msvc dll versioning/cp.bat

```diff
@@ -0,0 +1,10 @@
+@ECHO OFF
+@REM FIXME: remove hack when meson handles filenames correctly
+@REM replace '/' with '\', since copy does not like '/'.
+@REM Ideally, meson should pass us the filenames with '\', but
+@REM https://github.com/mesonbuild/meson/pull/737 was reverted.

```

Comment:
FWIW, I think we should remove that abomination and I'll deal with it on the Cerbero end. It won't break stable (1.8/1.10) Cerbero anymore because that installs its own (old version of) Meson anyway.